src/autolike.ts

```diff
@@ -0,0 +1,36 @@
+// clicks the awesome-button if lame-button is not in the 'selected' state
+function voteAwesome() {
+  if (
+    <HTMLElement>document.getElementsByClassName('lame-button selected')[0] ==
+      undefined &&
```

Comment:
Shouldn't have to check if it's `== undefined`, AFAIK 
```ts
if( foo === undefined ) {
...
}
```
is the same as
```ts
if( !foo ) {
...
}
```

Reply:
Won't !foo recognize other false-y values that match the condition such as `0` `null`, `''` (empty string)?

Reply:
Good enough reason for me